lib/monitor.js

```diff
@@ -0,0 +1,329 @@
+const fs = require("fs");
+const https = require("https");
+const { execSync } = require("child_process");
+const path = require("path");
+const url = require("url");
+const { log } = require("./logger");
+const {
+  CONFIGURATION_FILE_PATH,
+  GITHUB_SQL_ACCOUNT_REST_API_MONITOR_RELEASE_URL,
+  SQL_ACCOUNT_REST_API_MONITOR_APPLICATION_NAME,
+  WriteJsonFile,
+} = require("../data/constants");
+const { exit } = require("process");
+const axios = require("axios");
+
+// Check if the script is running as admin
+function isAdmin() {
```

Comment:
mấy cái function kiểu này hoàn toàn có thể reusable ở đâu đó (ví dụ trong libs, utils)